packages/server/src/app.ts

```diff
@@ -18,7 +18,9 @@ const { logAlert } = require("@budibase/backend-core/logging")
 const { Thread } = require("./threads")
 import redis from "./utilities/redis"
 import * as migrations from "./migrations"
-import { events, installation } from "@budibase/backend-core"
+import { events, installation, tenancy } from "@budibase/backend-core"
+import { createAdminUser, getChecklist } from "./utilities/workerRequests"
+import { DEFAULT_TENANT_ID } from "@budibase/backend-core/dist/src/constants"
```

Comment:
Can remove the `DEFAULT_TENANT_ID` import as its unused (and replaced by later comment).